### ISSUE TITLE
[WordPress Production-Ready] Deployment issue

### ISSUE THREAD

**Issue by KingMilli75** [bitnami/vms#785]
### Describe your issue as much as you can

Hi,

i tried 3 times to deploy a stack and now i am stuck with a error that i do not understand as its related to a template issue. 

error i receive:  

```
2023-03-21 16:58:48 UTC+0100	VPCStack	  DELETE_IN_PROGRESS
-
2023-03-21 16:58:48 UTC+0100	BastionStack  DELETE_COMPLETE
-
2023-03-21 16:56:54 UTC+0100	BastionStack	DELETE_IN_PROGRESS
-
2023-03-21 16:56:53 UTC+0100	WordPressStack	DELETE_COMPLETE
-
2023-03-21 16:56:41 UTC+0100	Kupholik	ROLLBACK_IN_PROGRESS
The following resource(s) failed to create: [WordPressStack]. Rollback requested by user.
2023-03-21 16:56:40 UTC+0100	WordPressStack CREATE_FAILED
Parameters: [DBStorageEncrypted] do not exist in the template
2023-03-21 16:56:38 UTC+0100	WordPressStack  CREATE_IN_PROGRESS
-
2023-03-21 16:56:35 UTC+0100	BastionStack	CREATE_COMPLETE
-
2023-03-21 16:49:05 UTC+0100	BastionStack	CREATE_IN_PROGRESS
Resource creation Initiated
2023-03-21 16:49:03 UTC+0100	BastionStack	CREATE_IN_PROGRESS
-
2023-03-21 16:49:00 UTC+0100	VPCStack	CREATE_COMPLETE
-
2023-03-21 16:45:29 UTC+0100	VPCStack	CREATE_IN_PROGRESS
Resource creation Initiated
2023-03-21 16:45:27 UTC+0100	VPCStack	CREATE_IN_PROGRESS
-
2023-03-21 16:45:21 UTC+0100	Kupholik	CREATE_IN_PROGRESS
User Initiated
```

Seems some error in the template if i understand this correctly i followed your guidelines:
[](https://docs.bitnami.com/aws-templates/apps/wordpress-production-ready/get-started/launch-production-ready-deployment/ )

**Comment by KingMilli75** [comment]
Btw it's on AWS

**Comment by jotamartos** [comment]
Hi @KingMilli75,

Sorry for the delay here. We tested the solution on our side and it passed the tests successfully. However, it seems there was a problem on the AWS side when copying the templates to the Marketplace and that's why the solution is failing during the launch process. 

However, we decided to drop support for this solution and the rest of available MultiTier solutions in the AWS Marketplace so we suggest you launch any other Bitnami WordPress solution in AWS. 

Regards.
